ChangeLog

```diff
@@ -30,6 +30,7 @@ ver. 0.9.5 (2016/XX/XXX) - wanna-be-released
    * New Actions:
      - action.d/firewallcmd-rich-rules and action.d/firewallcmd-rich-logging (gh-1367)
 - Enhancements:
+   * extremely speedup of all sqlite database operations (gh-1436)
```

Comment:
>    There are 3 pragmas in this FR, and temp_store currenly increase speed of
>    upgrade only
>    A reference to PR is in the same line (gh-1436) where described all this,
>    otherwise there is git blame, that can show all the changes;
I (as a user) would hate to be demanded to figure out what git, where
the PR, etc for every change whenever upgrading.  So all that doesn't
warrant opaque changelog msgs -- there should be some kind of a summary
on what was done, not only what was achieved since there might be
multiple things which could lead to speed up.  In this case it is the
memory journaling/caching, and thus should be at least briefly mentioned